bigframes/ml/llm.py

```diff
@@ -54,6 +54,13 @@
     _GEMINI_1P5_PRO_FLASH_PREVIEW_ENDPOINT,
 )
 
+_CLAUDE_3_SONNET_ENDPOINT = "claude-3-sonnet"
```

Comment:
Could you add all BQML-supported models?

Reply:
Sorry, I thought you were going to add in this PR itself. Are you going to send another PR for this?

Reply:
Yes. need to setup the connection for other regions.